terraform/terraform.tfvars.example

```diff
@@ -0,0 +1,27 @@
+# Example of terraform.tfvars
+# 
+# You can copy the contents of this file to terraform.tfvars and modify 
+# values for your own setup to operate with values different from the defaults
+# 
+# A terraform.tfvars will override any other defined variables such as the ones 
+# in *.auto.tfvars or defaults in variables.tf
+# https://opentofu.org/docs/language/values/variables/#variable-definitions-tfvars-files
+# https://opentofu.org/docs/language/values/variables/#variable-definition-precedence
+
+# Provider variables
+incus_host_name    = "incus"     # default: incus
+incus_host_scheme  = "https"     # default: https
+incus_host_address = "127.0.0.1" # default: 127.0.0.1
+incus_host_port    = 443         # default: 443
+incus_host_default = true        # default: true
```

Comment:
That doesn't look ideal. The default should be to use the local Incus server over the unix socket, not go over the network as going over the network would require a pre-trusted certificate.

This is what's causing the tests to fail now as Terraform is now trying to communicate with Incus through 127.0.0.1:443 which doesn't work.

Also, when accessing over HTTPS, a better example port would be 8443 as that's the port that Incus typically listens on.

Reply:
Thanks for pointing that out, I hadn't considered running it locally and communicating over a unix socket as that hasn't been an option for me. 

I read through the provider docs more closely again, updated the variable defaults, and I expect this to work better for the tests. I wasn't sure if I should leave the schema value in the `terraform.tfvars.example` file with `https` or if I should also have it match the default `unix`.

Reply:
I expect a majority of folks (mostly developers) will run against a local Incus daemon, so defaulting to the unix socket is preferable.